config/kubesan/openshift/scc.yaml

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: kubesan
+allowPrivilegedContainer: true
```

Comment:
I'm assuming we need root because the code that manages system volumes needs it?